umbraco-cloud/set-up/private-nuget-feed.md

```diff
@@ -0,0 +1,96 @@
+
+# Private Nuget Feed on Umbraco Cloud
+
+A private NuGet feed is a package repository that is only accessible to a specific group of users, rather than being publicly available.
+
+Private feeds are often used to host internal libraries or proprietary software within an organization.
+
+NuGet is a package manager for the Microsoft development platform, including .NET. It gives you the ability to add, remove, and update libraries and tools in Visual Studio projects.
+
+In this tutorial, we'll be covering how to set up a private NuGet feed with Umbraco Cloud. 
+
+## Prerequisite
+
+To follow along with this tutorial, you'll need the following tools:
+
+1. Visual Studio - https://visualstudio.microsoft.com/downloads/
+2. A NuGet server such as MyGet - https://www.myget.org/
+
+If you don't already have these tools installed, you can download Visual Studio from the Microsoft website. 
+You can also setup a server on MyGet following the above link. 
+
+Now you can go ahead an do the following steps.
+
+### Step 1: Create a NuGet package
+
+The first step is to create and publish a NuGet package using visual studio.
+
+This can be done by following the Microsoft documentation.
+
+https://learn.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-visual-studio?tabs=netcore-cli
```

Comment:
```suggestion
This can be done by following the [Microsoft documentation.](https://learn.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-visual-studio?tabs=netcore-cli)
```